src/test/scala-2.11/markets/participants/OrderCancelerSpec.scala

```diff
@@ -0,0 +1,73 @@
+package markets.participants
+
+import akka.actor.ActorSystem
+import akka.agent.Agent
+import akka.testkit.{TestProbe, TestActorRef, TestKit}
+
+import java.util.UUID
+
+import markets.{Cancel, Accepted}
+import markets.orders.limit.LimitAskOrder
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Imports should be sorted alphabetically](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/1653013744/issues/source?bid=2921361&fileBranchId=3037089#l10)